app/models/account/states.rb

```diff
@@ -98,6 +100,20 @@ module Account::States
       scheduled_for_deletion.where.has { deleted_at <= value.ago }
     end
 
+    scope :without_traffic_since_time_ago, ->(value = TIME_TO_BE_INACTIVE) do
```

Comment:
I'd like to unit test the changes in this file too 😄 It is tested now (all the options), but through the job test, not by unit test, so I want to add that 😄